src/test/ui/parser/unsafe-foreign-mod-2.stderr

```diff
@@ -6,15 +6,19 @@ LL | extern "C" unsafe {
    | |          |
    | |          expected `{`
    | help: `unsafe` must come before `extern "C"`: `unsafe extern "C"`
+
+error: extern block cannot be declared unsafe
```

Comment:
Oh, huh. @SparrowLii, what's the purpose of this PR if extern blocks cannot be declared unsafe?

Reply:
I guess this just is to continue parsing instead of giving up and bailing?

Reply:
In fact, I also have some doubts, the safety of foreign mod only affects whether an error is reported in the subsequent ast validity check.

Reply:
I think it's reasonable to avoid boiling here and move on